### DISCUSSION TITLE
Directing emu towards filtered <targets>

### DISCUSSION THREAD

**Discussion by DougPHynes** [QutEcoacoustics/emu#423] [Q&A]
I have a task that involves finding recordings with incorrect offsets, i.e., where offsets don't align with daylight saving time (yes, I live in a foolish place that switches their clocks twice a year 😒). Once I find those recordings, I can adjust the incorrect offsets using EMUs awesome `rename` command.  Is it possible to feed EMU an object/ls/dir that had filtered recordings based on the time stamps in their filenames (into other words, "find all .flac recursively between 20221106T020000-0300 and 20220312T02000-0300, then `rename` with `--new-offset`)? I can't seem to pull this off with the wildcard options, without moving around files by hand (into new directories). Another thought I had was to shape and filter by datetime in R (as I have limited PS skills and run EMU through R somehow)...

**Comment by atruskie** [comment]
Great idea. I've created https://github.com/QutEcoacoustics/emu/issues/424 to track this as a feature request.

In the meantime. yeah, use another language to script your execution of EMU. It will be a fair bit slower, but it will work.

In PowerShell (something like):

```pwsh
Get-Content file-list.txt | ForEach-Object { Emu.exe rename --new-offset "+10:00" $_ }
```

In R, it's a loop running system commands to run the Eme process
